src/components/Panels/PanelHeader.vue

```diff
@@ -0,0 +1,13 @@
+<template>
+    <div class="panel-header">
+        {{ props.headerTitle }}
+        <span class="fas fa-minus-square minimize"></span>
+        <span class="fas fa-external-link-square-alt maximize"></span>
+    </div>
+</template>
+
+<script lang="ts" setup>
+const props = defineProps({
+    headerTitle: { type: String, default: 'Panel Header' },
```

Comment:
Can we make the texts translatable from the beginning itself, it will be difficult to translate later

Reply:
yes sure, I will try to do it.

Reply:
Worked on it, can you please review it.

https://user-images.githubusercontent.com/61665451/175772655-4800403c-62c9-4b73-a1e5-67e7bc319296.mp4

Reply:
fantastic